auth-web/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth-web",
-  "version": "1.0.19",
+  "version": "1.0.20",
```

Comment:
This project is up to 1.1.0 already (and there's another PR taking it to 1.1.1).